specification/compute/resource-manager/Microsoft.Compute/2017-10-15-preview/swagger/computeArtifactPublishingService.json

```diff
@@ -0,0 +1,870 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ComputeArtifactPublishingServiceClient",
+    "description": "The Compute Artifact Publishing Service Resource Provider Client.",
+    "version": "2017-10-15-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
Patch support missing for shared vm image.

Reply:
We don't have a patch API now.